packages/zod-firebase/src/schema/schema-firestore-query-factory.ts

```
@@ -3,28 +3,44 @@ import { type AggregateSpec, getAggregateFromServer, getDocs } from '@firebase/f
 import { type MetaOutputOptions } from '../base'
 import { applyQuerySpecification, type QuerySpecification } from '../query'
 
-import { type CollectionSchema, type SchemaFirestoreQueryFactory, type SchemaQuery } from './types'
+import {
+  type CollectionSchema,
+  type SchemaFirestoreQueryFactory,
+  type SchemaQuery,
+  type SchemaQuerySpecification,
+} from './types'
```

ESLint

[error] Unable to resolve path to module './types'.

(import-x/no-unresolved)